templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_list_boilerplate.html.twig

```diff
@@ -220,4 +219,17 @@
 {% block javascripts %}
     {{ parent() }}
     {{ webpackBundle('procedure-administrationListBoilerplate.js') }}
+    <script  nonce="{{ csp_nonce('script') }}">
```

Comment:
Since moving the whole template to vue might be a bit much, we should move the code inside the script block to `administrationListBoilerplate.js`